_includes/code/graphql.get.consistency.mdx

```diff
@@ -0,0 +1,109 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId="languages">
+<TabItem value="graphql" label="GraphQL">
+
+```graphql
+{
+  Get {
+    Article (consistencyLevel: ALL) {
```

Comment:
I would not use ALL because this could be a really heavy operation.
I would instead use ONE like in the JS-example.

I would also use the same consistency level (ONE) for all clients (see Java and Go examples below) for consistency reasons

Reply:
Updated. Thanks 🙏